wiki/Tournaments/oPHF/2018/en.md

```diff
@@ -0,0 +1,277 @@
+---
+tags:
+- oPHF 2018
+- oPHF2018
+---
+# osu!Philippines Holiday Friendlies 2018
+
+![oPHF2018 Poster](hf2018poster.png)

```

Comment:
oPHF2018 Logo
Place the image into a subfolder named `img` and then rename the file `logo.png`